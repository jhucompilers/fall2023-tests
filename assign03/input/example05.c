int sum(int a, int b) {
  int result;
  result = a + b;
  return result;
}

int main(void) {
  int x;
  x = sum(3, 4);
  return x;
}
