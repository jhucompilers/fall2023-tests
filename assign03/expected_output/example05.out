0|sum|function|function (int, int) returning int
1|a|variable|int
1|b|variable|int
2|result|variable|int
0|main|function|function () returning int
2|x|variable|int
